Packs/GenericWebhook/Integrations/GenericWebhook/GenericWebhook.py

```diff
@@ -56,7 +57,12 @@ async def handle_post(
         credentials: HTTPBasicCredentials = Depends(basic_auth),
         token: APIKey = Depends(token_auth)
 ):
+    header_name = None
+    request_headers = dict(request.headers)
+    secret_header = (header_name or 'Authorization').lower()
```

Comment:
minor comment:
the header_name is only defined in line 71, so we need the secret_header to be defined also after
and you can delete the header_name definition in line 69, and revert line 74

Reply:
yes makes sense. Just did another commit/push